compensations/src/main/java/org/jboss/narayana/compensations/impl/remote/RemoteBAControler.java

```diff
@@ -105,4 +105,10 @@ public ParticipantManager enlist(Class<? extends CompensationHandler> compensati
         return new RemoteParticipantManager(pm);
     }
 
+    @Override
+    public ParticipantManager enlist(CompensationHandler compensationHandler, ConfirmationHandler confirmationHandler, TransactionLoggedHandler transactionLoggedHandler) throws Exception {
+
+        throw new RuntimeException("Not implemented.");
```

Comment:
It's not used in the example, so I left it unimplemented. In the real implementation this will me implemented properly.